src/config/dark-sites.config

```diff
@@ -636,6 +637,7 @@ pr0gramm.com
 pre.fyp.nl
 premid.app
 primevideo.com
+privacytools.io
```

Comment:
privacytools.io doesn't meet [the requirements](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#adding-a-website-that-is-already-dark).